### PR TITLE
3792/group library service routes into a controller

### PR DESCRIPTION
**Pull request by EVogel1999**
This PR adds docs to the library service routes and pulls out the routes from express route driver to the library controller.  If you would like to see these docs, rebuild the clark gateway image then go to http://localhost:3001/docs.  Please let me know if I missed anything, I was digging around library service to find missing information and couldn't find anything.  About the only thing I am unsure of is if we throw any 400 errors in library service.

Completes story [3792/group library service routes into a controller](https://app.clubhouse.io/clarkcan/story/3792/group-library-service-routes-into-a-controller).

### CODE REVIEW

**Review by jmulholland4014 — APPROVED** [review]

**Review by pzalep1 — APPROVED** [review]
Looks good!

**Review by mfranz1 — APPROVED** [review]

File: src/drivers/express/ExpressRouteDriver.ts [hunk 1]
```diff
@@ -312,15 +311,6 @@ export default class ExpressRouteDriver {
         }),
       );
 
-    router.get(
```
Comment (Cwagne17) [inline src/drivers/express/ExpressRouteDriver.ts:315]: isn't this learning objects not library service?

File: src/drivers/express/ExpressRouteDriver.ts [hunk 2]
```diff
@@ -332,16 +322,6 @@ export default class ExpressRouteDriver {
         },
       }));
 
-    // Retrieves the metrics for a learning object
-    router.get(
```
Comment (Cwagne17) [inline src/drivers/express/ExpressRouteDriver.ts:336]: Same here

File: src/drivers/express/ExpressRouteDriver.ts (hunk 1 above)
Comment (EVogel1999) [reply src/drivers/express/ExpressRouteDriver.ts:315]: It's proxying to the CART_API which is why it's grouped in the library service controller.

**Pull request merged**